0_RootFS/gcc_common.jl

```diff
@@ -211,14 +208,55 @@ function gcc_script(compiler_target::Platform)
+        #     --prefix=${prefix} \
+        #     --target=${COMPILER_TARGET} \
+        #     --host=${MACHTYPE} \
+        #     --with-libtapi=${prefix}
+
+        # Does not work:
+        #     CFLAGS="-isysroot ${sysroot}"
+        #     CXXFLAGS="-isysroot ${sysroot}"
+        #     CPPFLAGS="-isystem ${sysroot}/usr/include"
+        #     CPPFLAGS="-isystem /opt/x86_64-apple-darwin14/x86_64-apple-darwin14/sys-root/usr/include"
+        #     CPPFLAGS="-isystem /workspace/x86_64-apple-darwin14/destdir/x86_64-apple-darwin14/sys-root/System/Library/Frameworks/Kernel.framework/Versions/A/Headers"
+
+        # $sysroot:
+        #     /workspace/destdir/x86_64-apple-darwin14/sys-root
+        # find:
+        #     /opt/x86_64-apple-darwin14/x86_64-apple-darwin14/sys-root/usr/include/sys/cdefs.h
+        #     /opt/x86_64-apple-darwin14/x86_64-apple-darwin14/sys-root/System/Library/Frameworks/Kernel.framework/Versions/A/Headers/sys/cdefs.h
+        #     /workspace/srcdir/cctools-port/cctools/include/sys/cdefs.h
+        #     /workspace/srcdir/gcc-14.2.0/fixincludes/tests/base/sys/cdefs.h
+        #     /workspace/srcdir/MacOSX10.12.sdk/usr/include/sys/cdefs.h
+        #     /workspace/srcdir/MacOSX10.12.sdk/System/Library/Frameworks/Kernel.framework/Versions/A/Headers/sys/cdefs.h
+        #     /workspace/x86_64-apple-darwin14/destdir/x86_64-apple-darwin14/sys-root/usr/include/sys/cdefs.h
+        #     /workspace/x86_64-apple-darwin14/destdir/x86_64-apple-darwin14/sys-root/System/Library/Frameworks/Kernel.framework/Versions/A/Headers/sys/cdefs.h
+
+        # find / -name cdefs.h >CDEFS-LOCS
+        # /opt/x86_64-apple-darwin14/x86_64-apple-darwin14/sys-root/usr/include/sys/cdefs.h
+        # /opt/x86_64-apple-darwin14/x86_64-apple-darwin14/sys-root/System/Library/Frameworks/Kernel.framework/Versions/A/Headers/sys/cdefs.h
+        # /workspace/srcdir/cctools-port/cctools/include/sys/cdefs.h
+        # /workspace/srcdir/gcc-14.2.0/fixincludes/tests/base/sys/cdefs.h
+        # /workspace/srcdir/MacOSX10.12.sdk/usr/include/sys/cdefs.h
+        # /workspace/srcdir/MacOSX10.12.sdk/System/Library/Frameworks/Kernel.framework/Versions/A/Headers/sys/cdefs.h
+        # /workspace/x86_64-apple-darwin14/destdir/x86_64-apple-darwin14/sys-root/usr/include/sys/cdefs.h
+        # /workspace/x86_64-apple-darwin14/destdir/x86_64-apple-darwin14/sys-root/System/Library/Frameworks/Kernel.framework/Versions/A/Headers/sys/cdefs.h
+
+        ${WORKSPACE}/srcdir/cctools-port/cctools/configure \
             --prefix=${prefix} \
             --target=${COMPILER_TARGET} \
             --host=${MACHTYPE} \
-            --with-libtapi=${prefix}
-        make -j${nproc}
+            CC=/usr/bin/clang \
+            CXX=/usr/bin/clang++ \
+            CPPFLAGS="-I/workspace/x86_64-apple-darwin14/destdir/x86_64-apple-darwin14/sys-root/System/Library/Frameworks/Kernel.framework/Versions/A/Headers" \
```

Comment:
I'm moderately sure this is wrong: this is an executable for the host, not the target.

Reply:
Yes, this is where I'm stuck. Something isn't working, and I've been flailing adding various paths.